parts/inc/Sv_set

```diff
@@ -160,6 +160,16 @@ newSVsv_nomg(sv)
 
 #endif
 
+void
+sv_setsv_compile_test(sv)
+        SV *sv
+        CODE:
+                sv_setsv(sv, NULL);
+                sv_setsv_flags(sv, NULL, 0);
```

Comment:
This should fix Perl 5.6 compilation which does not have `sv_setsv_flags` macro.
```suggestion
#ifdef sv_setsv_flags
                sv_setsv_flags(sv, NULL, 0);
#endif
```

Reply:
excellent giving it a try